### ISSUE TITLE
Vue 3 upgrade - badge go live button does not get focus on cancel

### ISSUE THREAD

**Issue by dwalizer** [NationalSecurityAgency/skills-service#2635]
When hitting the Go Live button on the Badge card, pressing escape to cancel doesn't return focus to the button

**Comment by dwalizer** [comment]
Should be fixed in #2634

**Comment by sudo-may** [comment]
fixed via #2632
